pytket/phir/qtm_machine.py

```diff
@@ -27,16 +27,11 @@ class QtmMachine(Enum):
         size=20,
         gateset=QTM_DEFAULT_GATESET,
         tq_options={0, 2, 4, 6, 8, 10, 12, 14, 16, 18},
-        # need to get better timing values for below
-        # but will have to look them up in hqcompiler
-        timings=MachineTimings(tq_time=3.0, sq_time=1.0, qb_swap_time=2.0),
-    ),
-    QtmMachine.H1_2: Machine(
```

Comment:
I missed the rationale behind deleting this machine, do the same values apply for `H1_1` and `H1_2`? If so, does it make sense to rename the `H1_1` machine to just `H1`?

Reply:
the H1_1 and H1_2 machine objects were going to be identical within pytket-phir, so there was no reason to include 2 classes that would have exactly the same parameters

Reply:
it does make sense to just call it 'H1' in that case